rpm/sofia-sip.spec

```diff
@@ -1,17 +1,20 @@
 Name:       sofia-sip
 
 Summary:    Sofia SIP User-Agent library
-Version:    1.12.11
-Release:    2
-Group:      Communications/Telephony and IM
+Version:    1.13.4
```

Comment:
Please try adding this macro reduce the amount of changes needed in %files when the version number changes:

```rpm
'%global version_major_minor =
 %{lua: foo=string.gsub(rpm.expand("%version", ".%d$", ""); print(version_major_minor)}'
```

Reply:
Or would it make more sense to use wildcards in %files section to remove the need for version in there?

Reply:
Wildcards usage sounds simpler...

Reply:
Added a simple `ver` macro without any fancy stuff.

Reply:
No ver anymore, using wildcards